### PR TITLE
[IMP] partner_relations: merge from development

### PR DESCRIPTION
**Pull request by hbrunn**
This branch fixes two functional issues
- better check for overlapping relations
- edit relations within a popup

And improves some poorly phrased dutch translations

### CODE REVIEW

File: partner_relations/view/res_partner.xml [hunk 1]
```diff
@@ -79,6 +80,53 @@
...
+                                    <field
+                                        name="partner_id_display"
+                                        required="True"
+                                        attrs="{
+                                            'readonly': [('type_selection_id','=',False)],
+                                        }"
+                                        options="{'create': false, 'create_edit': false}"
+                                        widget="many2one_clickable"
+                                    />
+                                    <field
```
Comment (pedrobaeza) [inline partner_relations/view/res_partner.xml]: I think this style is not very readable. Don't you prefer for these cases with just one attribute to put them in only line?

**Comment by pedrobaeza** [comment]
One minor style issue. The rest LGTM.

File: partner_relations/view/res_partner.xml (hunk 1 above)
Comment (hbrunn) [inline partner_relations/view/res_partner.xml]: I leave this case to @NL66278

Comment (NL66278) [inline partner_relations/view/res_partner.xml]: Fields with really one attribute, that is only the name, I prefer on one line. When there are multiple attributes I prefer multiple lines, first to keep the line length within limits, secondly for easier adding, copying and deleting attributes. It certainly helps when using line oriented editors like vim.

Comment (pedrobaeza) [inline partner_relations/view/res_partner.xml]: I agree with you on multi-attributes tags, but for one attribute it's very ugly.

Comment (NL66278) [inline partner_relations/view/res_partner.xml]: @hbrunn Holger, I cannot change your pull request. Is it a problem for you to change the lines where the one-attribute elements are spread out unnecesarrily?

Comment (hbrunn) [inline partner_relations/view/res_partner.xml]: not at all, it's my pleasure @pedrobaeza the lines are fixed even though this doesn't show up here

**Comment by pedrobaeza** [comment]
Thanks for the changes.

:+1:

**Comment by guewen** [comment]
:+1:

**Comment by NL66278** [comment]
:+1:  LGTM

**Pull request merged**